editor/sidebar/last-revision/index.js

```diff
@@ -81,6 +81,10 @@ class LastRevision extends Component {
 		const { revisions } = this.state;
 		const lastRevision = revisions.length ? revisions[ 0 ] : null;
 
+		if ( revisions.length <= 0 ) {
+			return false;
```

Comment:
For an empty state, `null` might be the more appropriate return value, but this works just as well.